File: app/src/main/java/com/peakmain/basicui/activity/home/recylcer/activity/LinearGroupingActivity.java
package com.peakmain.basicui.activity.home.recylcer.activity;

import com.peakmain.basicui.R;

/**
 * author ：Peakmain
 * createTime：2020/3/23
 * mail:2726449200@qq.com
 * describe：
 */
public class LinearGroupingActivity extends BaseRecyclerAcitvity {

    @Override
    protected int getLayoutId() {
        return R.layout.basic_linear_recycler_view;
    }


    @Override
    protected void initData() {
        mNavigationBuilder.setTitleText("LinearLayoutManager实现分组").create();
        mRecyclerView.setAdapter(mGroupAdapter);
    }


}

File: app/src/main/java/com/peakmain/basicui/activity/home/recylcer/activity/LinearGroupingActivity.kt
package com.peakmain.basicui.activity.home.recylcer.activity

import com.peakmain.basicui.R

/**
 * author ：Peakmain
 * createTime：2020/3/23
 * mail:2726449200@qq.com
 * describe：
 */
class LinearGroupingActivity : BaseRecyclerAcitvity() {
    override fun getLayoutId(): Int {
        return R.layout.basic_linear_recycler_view
    }

    override fun initData() {
        mNavigationBuilder!!.setTitleText("LinearLayoutManager实现分组").create()
        mRecyclerView!!.adapter = mGroupAdapter
    }
}